NitroxInstallerActions/Patches/FirewallRules.cs

```diff
@@ -0,0 +1,158 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Diagnostics;
+using System.IO;
+
+namespace InstallerActions.Patches
+{
+    public class FirewallRules
```

Comment:
Please conform to our general style guidelines.  There should be a return between each method and between logical groupings of code.  In addition to that, we place spaces before and after each equal sign.  Please update the PR to conform to these.

Reply:
https://github.com/SubnauticaNitrox/Nitrox/blob/master/CONTRIBUTING.md guidelines in case you couldnt find them